firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -56,6 +57,9 @@
  * @hide
  */
 public class FirebaseInstallationServiceClient {
+
+  public static final int CREATE_INSTALLATION_TRAFFIC_STATS_TAG = 0x00008000;
```

Comment:
Thanks @regisd for fixing this up. I had some questions:

We have some other http requests that are made in this class. 

- Do we need stats_tag's for each of those as well?
- If yes, can we reuse the same stats tag? Or do we unique one's? And should we reserve them just like you did in cl/348055557

Like GenerateToken & Delete.

Reply:
Ah good point! Indeed, all http connections need to be tagged.

The initial cl/348055557 reserves a whole range in Firebase common. For more fine-grained tagging, we can use "Secondary tags".

Reply:
Added the secondary tags in cl/351113253. Since the primary tag is reserved for firebase-installations, I don't think there is a requirement in the order of submission of the CL and the PR.